demo/server/src/modules/datasync/MongoDataHandler.ts

```diff
@@ -0,0 +1,99 @@
+import { sync } from '@raincatcher/datasync-cloud';
+import { Db } from 'mongodb';
+
+/**
+ * Initializes global data handlers for feedhenry sync
+ */
+export class GlobalMongoDataHandler {
+  constructor(private db: Db) {
+  }
+
+  /**
+   * Init all handlers for CRUD operations
+   */
+  public initGlobalHandlers() {
+    this.handleList();
+    this.handleCreate();
+    this.handleSave();
+    this.handleCreate();
```

Comment:
maybe rename these to `setupHandle*` since they don't handle the events themselves, but setup the globals.